thirdparty/CMakeLists.txt

```diff
@@ -423,15 +437,16 @@ if(ENABLE_OV_PADDLE_FRONTEND OR ENABLE_OV_ONNX_FRONTEND OR ENABLE_OV_TF_FRONTEND
         if(CMAKE_VERBOSE_MAKEFILE)
             set(Protobuf_DEBUG ON)
         endif()
-        # Note: we also specify 'protobuf' in NAMES because of vcpkg
-        set(find_package_protobuf_names Protobuf protobuf)
+        if(OV_VCPKG_BUILD)
+            set(protobuf_config CONFIG)
+        endif()
         # try to find newer version first (major is changed)
         # see https://protobuf.dev/support/version-support/ and
         # https://github.com/protocolbuffers/protobuf/commit/d61f75ff6db36b4f9c0765f131f8edc2f86310fa
-        find_package(Protobuf 4.22.0 QUIET NAMES ${find_package_protobuf_names})
+        find_package(Protobuf 4.22.0 QUIET ${protobuf_config})
         if(NOT Protobuf_FOUND)
             # otherwise, fallback to existing default
-            find_package(Protobuf 3.20.3 REQUIRED NAMES ${find_package_protobuf_names})
+            find_package(Protobuf 3.20.3 REQUIRED ${protobuf_config})
```

Comment:
please, revert all changes in this file except VPUX -> NPU

Reply:
Should I revert the entire commit _Cherry-pick Support of protobuf >= 21_ https://github.com/openvinotoolkit/openvino/commit/f59ddf9a59c4c7877f40fdc5c14dacf9dec4d107 which updates 6 files? Or only the commit changes on this file `thirdparty/CMakeLists.txt`?

Reply:
Clarified. Changes reverted after rebase.